src/client-ocurl/opentelemetry_client_ocurl.ml

```diff
@@ -160,15 +159,7 @@ end = struct
       Pbrt.Encoder.to_string encoder
     in
 
-    let url =
-      let url = config.Config.url in
-      if url <> "" && String.get url (String.length url - 1) = '/' then
-        String.sub url 0 (String.length url - 1)
-      else
-        url
-    in
```

Comment:
This is now done when constructing a `config`: https://github.com/imandra-ai/ocaml-opentelemetry/pull/56/files#diff-94e77bde07a6cc16e48aafae28b4765617b765a4e9503dfd9902cd232d055105R50